index.d.ts

```diff
@@ -7,9 +7,13 @@
  * @param options all info for this parsing operation
  * @param variant the optional \uFE0F ("as image") variant, in case this info is anyhow meaningful. By default this is ignored.
  */
-declare type ParseCallback = (icon: string, options: object, variant: string) => string | false;
+type ParseCallback = (icon: string, options: object, variant: string) => string | false;
 
-declare interface TwemojiOptions {
+type ReplacerFunction = (substring: string, ...args: any[]) => string;
+
+type Replacer = string | ReplacerFunction;
```

Comment:
this is neither a long definition nor reused, could we inline it below for easier scanning? i.e.
```
replace(text: string, replacer: string | ReplacerFunction): string;
```

Reply:
done